apps/example/src/app/page.module.css

```diff
@@ -0,0 +1,168 @@
+.page {
```

Comment:
this file is probably not needed, assuming we are using Emotion in this project?